alws/crud/build_node.py

```diff
@@ -298,8 +300,41 @@ def append_errata_package(_, errata_package, artifact, rpm_info):
             )
         rpms.append(artifact)
 
+    def is_source_rpm_included(
```

Comment:
Since the function works not only with SRPMs, maybe change it's name accordingly?